cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Conversion.java

```diff
@@ -181,6 +181,7 @@ public Network convert(Reporter reporter) {
             convert(cgmes.busBarSections(), bbs -> new BusbarSectionConversion(bbs, context));
         }
 
+        convert(cgmes.grounds(), g -> new GroundConversion(g, context));
```

Comment:
After a talk with @flo-dup, this feature can add many buses in the bus/breaker view coming from the node/breaker view. To avoid performance issue, I think we have to test: the difference of bus number with an without the feature on a real network. If the difference is huge, a parameter must be added, to false by default.